spec/helpers/users_circuitverse_helper_spec.rb

```
@@ -64,56 +64,22 @@
-        @project.circuit_preview.purge
-        expect(Flipper.enabled?(:active_storage_s3, @user)).to be true
-        default_image_path = send(:return_circuit_preview, @project)
-        expect(default_image_path).to eq("/images/empty_project/default.png")
-        expect(project_image_preview(@project, @user)).to eq("/images/empty_project/default.png")
-      end
+    before do
+      @project = FactoryBot.create(:project)
+      @project.circuit_preview.attach(
+        io: Rails.root.join("spec/fixtures/files/default.png").open,
+        filename: "preview_1234.jpeg",
+        content_type: "img/jpeg",
```

RuboCop (1.75.5)

[convention] Avoid comma after the last parameter of a method call.

(Style/TrailingCommaInArguments)